pkg/cmd/show.go

```diff
@@ -269,7 +266,7 @@ func showPrometheus(targetReader TargetReader) {
 func showAltermanager(targetReader TargetReader) {
 	username, password = getMonitoringCredentials()
 	showPod("alertmanager", "seed", targetReader)
-	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "alertmanager", "-n", getTechnicalID())
+	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "alertmanager", "-n", getFromTargetInfo("project"))
```

Comment:
this is not correct

Reply:
https://github.com/gardener/gardenctl/issues/350

Reply:
```suggestion
	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "alertmanager", "-n", getFromTargetInfo("shootTechnicalID"))
```
even though it might get removed with #350, please do not commit a change that is clearly using the wrong namespace

Reply:
change back